src/amo/components/StaticPages/About/index.js

```diff
@@ -18,12 +18,22 @@ export class AboutBase extends React.Component {
   render() {
     const { i18n } = this.props;
 
-    const helpText = i18n.sprintf(i18n.gettext(`Help improve this website. It's open source, and you can file bugs and submit patches. <a href="%(url)s">GitHub</a> contains all of our current bugs, legacy bugs can still be found in Bugzilla.`),
-      { url: 'https://github.com/mozilla/olympia/issues' });
-    const blogText = i18n.sprintf(i18n.gettext(`Our <a href="%(url)s">Add-ons Blog</a> is regularly updated with information for both add-on enthusiasts and developers.`),
-      { url: 'http://blog.mozilla.com/addons/' });
-    const forumText = i18n.sprintf(i18n.gettext(`Our <a href="%(url)s">forums</a> are a good place to interact with the add-ons community and discuss upcoming changes to AMO.`),
-      { url: 'https://discourse.mozilla-community.org/c/add-ons' });
+    const helpText = i18n.sprintf(i18n.gettext(`Help improve this website.
```

Comment:
For context, and since there's a lot of these, I think it might be helpful to put them inline rather than have these intermediate `const`s.